varya/assets/sass/blocks/utilities/_editor.scss

```diff
@@ -158,6 +165,21 @@
 }
 
 /* Block Alignments */
+.wp-block {
+
+	// Gutenberg injects a rule that limits the max width of .wp-block to 580px
+	// This line overrides it to use the responsive spacing rules for default width content
+	&:not(.alignwide):not(.alignfull):not([data-align="full"]):not([data-align="wide"]){
```

Comment:
Ooof. Is there any way to fix this without all of those `:not()` selectors?

Reply:
I pushed a change that I believe fixes the issue, without using the `:not()` selectors.